ui/src/components/MainContainer.js

```diff
@@ -247,6 +248,12 @@ const MainContainer = () => {
           permissions={['admin', 'stewardship']}
           component={AdminUser}
         />
+        <PrivateRoute
+          exact
+          path="/admin/followUp"
```

Comment:
is it better if the path name is `/admin/followUp/baptism`? since this is specifically for baptism only

Reply:
Cab do once we add the other follow up pages